tools/wptserve/wptserve/server.py

```diff
@@ -168,7 +168,8 @@ def __init__(self, server_address, request_handler_cls,
             Server.config = config
         else:
             self.logger.debug("Using default configuration")
-            Server.config = {"host": server_address[0],
+            Server.config = {"browser_host": server_address[0],
```

Comment:
This doesn't seem very meaningful, to put `browser_host` and `server_host` and set them to o the same value.